src/main/java/org/broadinstitute/barclay/argparser/CommandLineProgramProperties.java

```diff
@@ -27,5 +26,6 @@
      */
     String usageExample() default "The author of this program hasn't included any example usage, please complain to them.";
```

Comment:
That's right, historically both GATK3 and Picard have only included a usage example as part as the general tool description (if at all). I love the idea of providing a usage example through an explicit declaration that can be rendered in the appropriate syntax. 

I agree it can be a followup ticket too.